### PR TITLE
Enlarge volume size in test_backup_status_for_unavailable_replicas

### PR DESCRIPTION
**Pull request by derekbit**
Enlarge volume size for increasing backup time

Longhorn/longhorn#5189

Signed-off-by: Derek Su <derek.su@suse.com>

### CODE REVIEW

**Comment by derekbit** [comment]
https://ci.longhorn.io/job/private/job/longhorn-tests-regression/2908

**Review by shuo-wu — APPROVED** [review]

File: manager/integration/tests/test_basic.py [hunk 1]
```diff
@@ -544,24 +544,25 @@ def test_backup_status_for_unavailable_replicas(set_random_backupstore, client,
...
 
     # write data to the volume
-    data = {
-        'pos': 0,
-        'content': common.generate_random_data(int(size)),
-    }
-    write_volume_data(volume, data)
+    volume_endpoint = get_volume_endpoint(volume)
+
+    # Take snapshots4 without overlapping
```
Comment (c3y1huang) [inline manager/integration/tests/test_basic.py:562]:
```suggestion
    # Take snapshots without overlapping
```

**Review by c3y1huang — CHANGES_REQUESTED** [review]

File: manager/integration/tests/test_basic.py (hunk 1 above)
Comment (derekbit) [reply manager/integration/tests/test_basic.py]: Fixed. Thank you.

**Review by c3y1huang — APPROVED** [review]

**Comment by yangchiu** [comment]
@Mergifyio backport v1.4.x

**Pull request merged**